pipeline/models/speciesModelRuns.R

```diff
@@ -68,7 +68,7 @@ for (i in 1:length(names(workflowList))) {
                                     safe = TRUE))
   
   # Run model (this directly saves output to folder specified above)
-  sdmWorkflow(workflow)
+  sdmWorkflow(workflow, predictionDim = c(300,400))
```

Comment:
is it more straightforward that this would use the 'res' specified in masterScript.R?
alternatively, should the desired prediction resolution (eg c(300,400)) be the target resolution for all environmental data?
regardless, maybe this parameter should also be moved to masterScript.R.